pkgs/development/python-modules/numpy/default.nix

```diff
@@ -40,25 +41,20 @@ let
   };
 in buildPythonPackage rec {
   pname = "numpy";
-  version = "1.19.4";
+  version = "1.20.0";
   format = "pyproject.toml";
+  disabled = pythonOlder "3.7";
```

Comment:
Should we keep an old version around for Python 2.7?

Reply:
There's already a `numpy/1.16.nix` in the directory, I think for that purpose. Do you not think that's enough?

Reply:
Python 2.7 is EOL and shouldn't be considered anymore unless it is absolutely necessary.